dashboard/app/models/course_version.rb

```diff
@@ -197,6 +197,7 @@ def summarize_for_assignment_dropdown(user, locale_code)
         is_stable: stable?,
         is_recommended: recommended?(locale_code),
         locales: content_root.supported_locale_names,
+        locale_codes: content_root.supported_locale_codes,
```

Comment:
We also want the locale codes associated with course information so we can filter them a little more easily than the localized terms for the languages. (locales is stuff like "Español" rather than "es-ES")